testdata/make_coverage_out

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Add the .sh extension to make it clearer? `make_coverage_out.sh`

Reply:
That would contradict our own styleguide (https://google.github.io/styleguide/shellguide.html#s2.1-file-extensions): "Executables should have no extension (strongly preferred)"